sale_comment_template/models/__init__.py

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
+#
+#
+# Copyright Camptocamp SA
+# author nbessi
+#
+#
```

Comment:
Use this template for init files: https://github.com/OCA/maintainer-tools/blob/c73e7d8ca936cce206256935ec20ad7083b7f176/template/module/__init__.py#L1-L2